bazel/repository_locations.bzl

```diff
@@ -878,13 +890,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Bazel rust rules",
         project_desc = "Bazel rust rules (used by Wasm)",
         project_url = "https://github.com/bazelbuild/rules_rust",
-        version = "7b8219a1955ebd599c275bd292d8153e5616d6a4",
-        sha256 = "426a7712af597d90301dcc29e63d62de5c2e09fb347692e89abb775ec97c74fe",
+        version = "aa7c6938cf1cc2973bc065c7532f89874bf09818",
+        sha256 = "5cb2fbcc3debebc7b68f5f66c1b7ef741bdcca87c70594de688d4518538c36c8",
         strip_prefix = "rules_rust-{version}",
         urls = ["https://github.com/bazelbuild/rules_rust/archive/{version}.tar.gz"],
         use_category = ["dataplane_ext"],
         extensions = ["envoy.wasm.runtime.wasmtime"],
```

Comment:
Is this list of extensions complete? The other dependencies from this Github org have a more comprehensive list - https://github.com/envoyproxy/envoy/blob/master/bazel/repository_locations.bzl#L820-L865

Reply:
Yes, rust is only used by wasmtime runtime.

Reply:
Note: this list is referring to `rules_rust` and not to `proxy_wasm_rust_sdk`.